### PR TITLE
fix(composer): install PHP Redis default version

### PR DESCRIPTION
**Pull request by EtienneM**
Deployment currently fails with:

```
Root composer.json requires PHP extension ext-redis ^3.1 but it has the wrong version installed (6.1.0).
```

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
     "require": {
         "php": "^8.1",
         "cron/cron": "^1.8.0",
-        "ext-redis": "^3.1"
+        "ext-redis": "*"
```
Comment (sc-david-voisin) [inline composer.json:6]: I prefer restrictive versions, why to allow them all here ?

File: README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 This application is a simple PHP application using the package `cron/cron` installed from Composer
 
-Application → https://sample-php-cron.scalingo.io/
+Application → https://php-cron.is-easy-on-scalingo.com
```
Comment (sc-david-voisin) [inline README.md:6]: question: I checked https://sample-php-cron.scalingo.io/ redirects to https://php-cron.is-easy-on-scalingo.com which is fine ^^; how to know about which such URLs are the old ones and the new ones to use ?

Comment (EtienneM) [reply README.md:6]: The new one is the one defined as he canonical domain on the Scalingo applcation

File: composer.json (hunk 1 above)
Comment (EtienneM) [reply composer.json:6]: This is a feature, or a lack of feature, of our PHP buildpack. Related issue: https://github.com/Scalingo/php-buildpack/issues/395

**Review by sc-david-voisin — APPROVED** [review]

**Pull request merged**